CLDR-18574 Hub update for ISO8601 (#4672)
unicode-org/cldr: docs/site/translation.md

<!-- I don't think we want to encourage this: ... if there is a need to have coverage at lower level in some locale,
please file a ticket. [CLDR-18283](https://unicode-org.atlassian.net/browse/CLDR-18283) -->

#### Gregorian Calendar (Year First) calendar

This is a variant of the Gregorian calendar whose formats always use year-month-day ordering and a 24-hour time cycle.
*Note: the code is `iso8601`, but disregard that; it will be changed after submission.*

Page | Code | English
-|-|-
Date & Time / Gregorian YMD | yMMMd | y MMM d
Date & Time / Gregorian YMD | … | …
Locale Display Names / Keys | calendar-iso8601 | Gregorian Calendar (Year First)

Please go though the Gregorian Calendar (Year First) fields.
* You should change separators to match what is acceptable in your language. 

> * The ordering of fields in the pattern should be strictly the following order (for any that occur in a particular pattern).

Reviewer: ```suggestion
* The ordering of fields in the pattern should be strictly the following order (for any that occur in a particular pattern):
```

Author: Fixed this one (text was different so I couldn't commit the suggestion).

It can be tricky to understand the difference, so be sure to carefully read [Rational Formatting](https://cldr.unicode.org/translation/number-currency-formats/number-and-currency-patterns#rational-formatting) before making any changes.

<!-- [CLDR-17570](https://unicode-org.atlassian.net/browse/CLDR-17570) -->

Here are the the English values and a short description of their purpose:

Code | Default Value | Description
-|:-:|-
`Rational` | {0}⁄{1} | The format for a rational fraction with arbitrary numerator and denominator; the English pattern uses the Unicode character ‘⁄’ U+2044 FRACTION SLASH which causes composition of fractions such as <sup>22</sup>⁄<sub>7</sub>.
`Integer + Rational` | {0} {1} | The format for combining an integer with a rational fraction composed using the pattern above; the English pattern uses U+202F NARROW NO-BREAK SPACE (NNBSP) to produce a `non-breaking thin space`.
`Integer + Rational-superSub` | {0}⁠{1} | The format for combining an integer with a rational fraction using composed using the pattern above; the English pattern uses U+2060 WORD JOINER, a zero-width no-break space.

> `Usage` | sometimes | An indication of the extent to which rational fractions are used in the locale; may be one of `never` or `sometimes`

Reviewer: ```suggestion
`Usage` | sometimes | An indication of the extent to which rational fractions are used in the locale; must be either `never` or `sometimes`.
```
